pages/dynamic-portal.tsx

```diff
@@ -105,9 +105,9 @@ const DynamicTemplates: NextPageWithLayout<Props> = ({
       customOptionsConfig(customField.enumOptions)),
   };
 
-  const publishableKey = process.env.NEXT_PUBLIC_PUBLISHABLE_KEY;
+  const publishableKey = process.env.NEXT_PUBLIC_FLATFILE_PUBLISHABLE_KEY;
```

Comment:
Updated this to be consistent with the other apps.

Reply:
Make sure to add this new one to heroku and deploy before deleting the old